custom_components/kia_uvo/services.py

```diff
@@ -262,6 +264,12 @@ async def async_handle_stop_valet_mode(call):
         vehicle_id = _get_vehicle_id_from_device(hass, call)
         await coordinator.async_stop_valet_mode(vehicle_id)
 
+    async def async_handle_set_vehicle_to_load_discharge_limit(call):
+        coordinator = _get_coordinator_from_device(hass, call)
+        vehicle_id = _get_vehicle_id_from_device(hass, call)
+        await coordinator.async_set_vehicle_to_load_discharge_limit(vehicle_id)
+        v2l_discharge_limit = call.data.get("v2l_limit")
```

Comment:
This line would move above the action of the coordinator.  As per this line that method expects vehicle_id, followed by the value.  Value being v2l_discharge_limit.

https://github.com/Hyundai-Kia-Connect/kia_uvo/blob/20c7f3233347278e7fdc9b18c71960373661c3fe/custom_components/kia_uvo/coordinator.py#L330